mne/evoked.py

```diff
@@ -681,7 +681,7 @@ class EvokedArray(Evoked):
     """
 
     @verbose
-    def __init__(self, data, info, tmin, comment='', nave=1, kind='average',
+    def __init__(self, data, info, tmin=0., comment='', nave=1, kind='average',
```

Comment:
in `tests/test_evoked.py` can you modify a test to make use of this new default?